apps/test-suite/tests/SQLite.ts

```diff
@@ -6,6 +6,8 @@ import * as SQLite from 'expo-sqlite';
 
 export const name = 'SQLite';
 
+const VERSION = '3.39.2';
```

Comment:
please add a comment both here and podspec/gradle that the version should be aligned.